src/client/components/status/status.js

```diff
@@ -41,7 +44,7 @@ class Status extends React.Component {
     }
 
     return <span />;
-  }
+  } 
 }
-
-export default Status;
\ No newline at end of file
+export default Status;
+export {passedexp};
```

Comment:
I don't really understand why you need to export this property and fill it during render. This seems to be hacky.

 The players who passed are stored in `G`, right? Can't you just add a new property to `<Leaderboard />` and pass `G` to it when calling the component in `<Sidebar />`